src/app/components/events/create-event/create-event.component.html

```diff
@@ -74,61 +67,80 @@
         >
       </mat-select>
     </mat-form-field>
-    <br />
 
-    <mat-form-field appearance="outline">
+    <div class="section-title">Time</div>
+    <mat-form-field appearance="outline" color="accent">
       <mat-label>Starts At</mat-label>
       <input
         matInput
-        [matDatepicker]="startsAtPicker"
+        [ngxMatDatetimePicker]="startsAtPicker"
+        [min]="minStartsAtDate"
         name="startsAt"
         placeholder="Select a date and time"
-        [(ngModel)]="formData.startsAt"
+        formControlName="startsAt"
         required
       />
       <mat-datepicker-toggle
         matSuffix
-        [for]="startsAtPicker"
+        [for]="$any(startsAtPicker)"
       ></mat-datepicker-toggle>
-      <mat-datepicker #startsAtPicker></mat-datepicker>
+      <ngx-mat-datetime-picker #startsAtPicker></ngx-mat-datetime-picker>
     </mat-form-field>
-    <br />
-
-    <mat-form-field appearance="outline">
+    <mat-form-field appearance="outline" color="accent">
       <mat-label>Ends At</mat-label>
       <input
         matInput
-        [matDatepicker]="endsAtPicker"
+        [ngxMatDatetimePicker]="endsAtPicker"
         name="endsAt"
+        [min]="eventForm.get('startsAt')?.value"
         placeholder="Select a date and time"
-        [(ngModel)]="formData.endsAt"
+        formControlName="endsAt"
         required
       />
       <mat-datepicker-toggle
         matSuffix
-        [for]="endsAtPicker"
+        [for]="$any(endsAtPicker)"
       ></mat-datepicker-toggle>
-      <mat-datepicker #endsAtPicker></mat-datepicker>
+      <ngx-mat-datetime-picker #endsAtPicker></ngx-mat-datetime-picker>
     </mat-form-field>
-    <br />
 
-    <mat-form-field appearance="outline">
+    <div class="bets-container">
+      <div class="section-title" style="margin-bottom: 2rem" id="bet-section">
```

Comment:
no inline styles please. If you don't use primeflex, you could add some general styling classes in your styles.css file and use them in your components without defining them locally. That would be smth like : .flex class(for display flex), margins and paddings classes, etc